housekeeping/src/main/java/com/hotels/housekeeping/model/HousekeepingLegacyReplicaPath.java

```diff
@@ -32,15 +32,20 @@
   protected HousekeepingLegacyReplicaPath() {}
 
   public HousekeepingLegacyReplicaPath(String path) {
-    this.path = path;
-    eventId = "";
-    pathEventId = "";
+    this("", "", path, null, null);
   }
 
-  public HousekeepingLegacyReplicaPath(String eventId, String pathEventId, String path) {
+  public HousekeepingLegacyReplicaPath(
```

Comment:
This and the changes to the `LegacyReplicaPath` are effectively backwards incompatible changes and these public methods/classes are part of the "API" we use externally so I'm now thinking we should go for a major version bump.

Reply:
It's only additions right? I can add another constructor if we want that? Not too fussed about major version as well, up to you.

Reply:
Well, the previous method here which took 3 arguments is now gone right? So if anyone was calling it their code will break.  Same for anyone implementing the interface below. This is largely academic since we're the users of it but since these are public parts of the library I'd prefer a major bump.

Reply:
yes except the constructor,as I said. major bumping it no worries